frontend/integration-tests/tests/secrets.scenario.ts

```diff
@@ -118,4 +118,123 @@ describe('Interacting with the create secret forms', () => {
       await crudView.deleteResource('secrets', 'Secret', sshSourceSecretName);
     });
   });
+
+  describe('Registry credentials image secrets', () => {
+    const credentialsImageSecretName = 'registry-credentials-image-secret';
+    const credentialsImageSecretAddress = 'https://index.openshift.io/v';
+    const credentialsImageSecretAddressUpdated = 'https://index.openshift.io/updated/v1';
+    const credentialsImageSecretUsername = 'username';
+    const credentialsImageSecretUsernameUpdated = 'usernameUpdated';
+    const credentialsImageSecretPassword = 'password';
+    const credentialsImageSecretPasswordUpdated = 'passwordUpdated';
+    const credentialsImageSecretMail = 'test@secret.com';
+    const credentialsImageSecretMailUpdated = 'testUpdated@secret.com';
+    const credentialsToCheck = {
+      '.dockerconfigjson': {
+        'auths':{
+          'https://index.openshift.io/v0':{
+            'username':'username0',
+            'password':'password0',
+            // base64 encoded 'username0:password0' value
+            'auth':'dXNlcm5hbWUwOnBhc3N3b3JkMA==',
```

Comment:
I think it could be better to do something like:

```js
auth: `btoa('username0:password0'),
```

or maybe

```js
const encode = (username, password) => btoa(`${username}:${password}`);

const username0 = 'username0';
const password0 = 'password0';
// ...
username: username0,
password: password0,
auth: encode(username0, password0),
```

Then you know the value is right and don't need the comment explaining.